src/components/Navigation/NavigationItem/NavigationItem.tsx

```diff
@@ -60,10 +62,15 @@ const NavigationItem = forwardRef<HTMLLIElement, NavigationItemProps>(
     },
     ref,
   ) => {
+    const { closeMenu } = React.useContext(HeaderContext);
```

Comment:
Can this be moved to the src/components/Navigation/NavigationItemTrigger/NavigationItemTrigger.tsx so all the close logic is being added to the onClick